lib/src/contextual-menu/ItemAction.tsx

```diff
@@ -66,11 +66,12 @@ const Action = styled.button<{ depthLevel: ItemActionProps["depthLevel"]; select
 
 const Icon = styled.span`
   display: flex;
-  font-size: 16px;
+  font-size: ${({ theme }) => theme.iconSize};
+  color: ${({ theme }) => theme.iconColor};
 
   svg {
-    height: 16px;
-    width: 16px;
+    height: ${({ theme }) => theme.iconSize};
+    width: ${({ theme }) => theme.iconSize};
```

Comment:
We can safely remove these styles since they will not be a `SVG`

Reply:
We cannot remove this styles!